webapp/channels/src/components/integrations/add_incoming_webhook/add_incoming_webhook.test.tsx

```diff
@@ -26,16 +63,29 @@ describe('components/integrations/AddIncomingWebhook', () => {
     });
 
     test('should have called createIncomingHook', () => {
-        const hook = TestHelper.getIncomingWebhookMock({
-            channel_id: 'channel_id',
+        const hook = {
```

Comment:
Why do we remove the call to the TestHelper?

Reply:
This call was returning an object with some additional properties like `create_at` or `delete_at` that I assumed they are not relevant in this test. Do you think this call should stay?

Reply:
If I am not mistaken, it should stay, yes. The idea of this call is to mock the default values so it looks like the object you would naturally receive out in the wild. If those extra values are creating any issue in terms of types or anything, we can look a bit deeper why that is, but it is better if we use it.

Reply:
Ok, I get it :) I've added these additional properties. Now the test is comparing whole objects, and it seems to work :+1:.